pkg/controllers/c2cc/probe.go

```
@@ -0,0 +1,228 @@
+		}
+	}()
+
+	pm := &probeManager{
+		client: msClient,
+		probes: make(map[string]context.CancelFunc),
+	}
+
+	factory := microshiftinformers.NewSharedInformerFactory(msClient, informerResync)
+	informer := factory.Microshift().V1alpha1().RemoteClusters().Informer()
+
+	informer.AddEventHandler(cache.ResourceEventHandlerFuncs{
```

golangci-lint (2.12.2)

[error] Error return value of `informer.AddEventHandler` is not checked

(errcheck)